### PR TITLE
Port nextpage block to the ReactNative mobile app

### PR DESCRIPTION
**Pull request by pinarol**
## Description
This PR contains the minimum set of required changes for ReactNative mobile app be able to use the nextpage block. Changes can be tested using this [PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/209).

## How has this been tested?

- [x]  Make sure the (web) GB build and webapp work normally
- [x]  Make sure the GB mobile app works normally and nextpage block is available for adding.

## Types of changes
- Nextpage block is added to the list of registered blocks for mobile app
- Edit function is extracted from index.js and put into separate files for mobile(edit.native.js) and web(edit.js)
- New .scss is added for mobile: editor.native.scss

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code has proper inline documentation.

### CODE REVIEW

File: packages/notices/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
 		"build-module"
 	],
 	"main": "build/index.js",
+	"react-native": "src/index",
```
Comment (gziolo) [inline packages/notices/package.json]: 👍 we missed that, sorry.

File: packages/block-library/src/nextpage/editor.native.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// @format
+
+.block-library-nextpage__container {
+	align-items: center;
+	padding-left: 4;
```
Comment (gziolo) [inline packages/block-library/src/nextpage/editor.native.scss]: Does RN version of sass support shortcuts present in CSS like `padding: 4px`?

@hypest this is confusing to have something which looks like CSS but isn't. Maybe we should revisit this approach and start using JS files with style objects instead?

File: packages/block-library/src/nextpage/edit.native.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import styles from './editor.scss';
+
+export default function NextPageEdit( { attributes } ) {
+	const { customText } = attributes;
```
Comment (gziolo) [inline packages/block-library/src/nextpage/edit.native.js]: This three lines can be combined into one as follows:

```js
const { customText = __( 'Page break' ) } = attributes;
```
:)

**Review by gziolo — APPROVED** [review]
This PR looks good. I don't see any blockers. I left one comment where you can shorten the code.

In particutlar `const value = customText !== undefined ? customText : defaultText;` is something that is very hard to read. You usually will see it expressed as `const value = customText || defaultText;`

File: packages/block-library/src/nextpage/editor.native.scss (hunk 2 above)
Comment (pinarol) [reply packages/block-library/src/nextpage/editor.native.scss]: padding: 4px worked! 👍

Comment (hypest) [reply packages/block-library/src/nextpage/editor.native.scss]: > Does RN version of sass support shortcuts present in CSS like padding: 4px

I think it does, one just needs to provide all 4 or more of the individual fields. Example: `padding: 4px 4px 4px 4px`.

> @hypest this is confusing to have something which looks like CSS but isn't. Maybe we should revisit this approach and start using JS files with style objects instead?

I think we haven't really put any new effort on the styling side of things so, let's defer this convo for a bit. I think we will do style related work later in the year.

File: packages/block-library/src/nextpage/editor.native.scss [hunk 4]
```diff
@@ -2,8 +2,5 @@
 
 .block-library-nextpage__container {
 	align-items: center;
-	padding-left: 4;
-	padding-right: 4;
-	padding-top: 4;
-	padding-bottom: 4;
+	padding: 4px 4px 4px 4px;
```
Comment (gziolo) [inline packages/block-library/src/nextpage/editor.native.scss]: In real CSS it can be even shorter: `padding: 4px;`

**Comment by gziolo** [comment]
Congrats 💯

**Pull request merged**